### PR TITLE
Redux state changes to support RDF generation

### PR DESCRIPTION
**Pull request by jermnelson**
The first steps for RDF generation in the Linked Data editor involves adding the following properties to the Redux state:

- `rdfPredicate` property set for propertyTemplates

### CODE REVIEW

File: src/reducers/index.js [hunk 1]
```diff
@@ -38,7 +38,8 @@ export const populatePropertyDefaults = (propertyTemplate) => {
   if (propertyTemplate === undefined || propertyTemplate === null || Object.keys(propertyTemplate).length < 1) {
     return defaults
   }
-  if (propertyTemplate.valueConstraint.defaults && propertyTemplate.valueConstraint.defaults.length > 0) {
+
+  if (propertyTemplate?.valueConstraint?.defaults && propertyTemplate.valueConstraint.defaults.length > 0) {
```
Comment (mjgiarlo) [inline src/reducers/index.js]: Because you're using the safe navigation operator here, you can drop the first condition and shorten this to:

```javascript
if (propertyTemplate?.valueConstraint?.defaults?.length > 0) {
```

Safe navigation will return `null` anywhere along the chain that an attribute isn't defined, and `null` is not greater than 0, so the condition does what we want.

File: src/reducers/inputs.js [hunk 2]
```diff
@@ -21,8 +21,11 @@ export const setMyItems = (state, action) => {
   reduxPath.reduce((obj, key) => {
     level++
     if (level === reduxPath.length) {
-      if ((key in obj) != true || !Object.keys(obj[key]).includes('items')) {
-        obj[key] = { items: [] }
+      if (!(key in obj)) {
+        obj[key] = {}
+      }
+      if (!Object.keys(obj[key]).includes('items')) {
```
Comment (mjgiarlo) [inline src/reducers/inputs.js]: Hmm, it may be JavaScript newbness, but it seems we're using two different strategies three lines apart to determine if an object contains a key. Would `if (!('items' in obj[key])) {` work here?

**Comment by jermnelson** [comment]
I'm closing this draft PR but leaving the feature branch for now in case anyone wants to reference it later.
